src/components/Header/Search/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useCallback } from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import _ from "lodash";
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import styles from "./Search.module.scss";
+
+const Search = () => {
+  const cards = useSelector((state) => Object.values(state.cards));
+  const [resultSet, setResultSet] = useState([]);
+  const [activeDropdown, setActiveDropdown] = useState(false);
+  const [inputText, setInputText] = useState("");
+
+  const doesExistsValueInString = (value, currentSearch) => {
+    const lowerCaseSearch = currentSearch.toLowerCase();
+    const valuePosition = lowerCaseSearch.search(value);
+    return valuePosition !== -1;
```

Comment:
Its easier to read when there is a line before/after (depends on the context) the `return`, in this case try something like
```suggestion
    const valuePosition = lowerCaseSearch.search(value);

    return valuePosition !== -1;
```